apps/explorer/lib/explorer/graphql.ex

```diff
@@ -316,11 +320,27 @@ defmodule Explorer.GraphQL do
   end
 
   def celo_tx_transfers_query do
+    token_contract_names = Util.get_token_contract_names()
+    token_symbols = Util.get_token_contract_symbols()
+
     from(
       tt in TokenTransfer,
       join: t in CeloParams,
       where: tt.token_contract_address_hash == t.address_value,
-      where: t.name == "goldToken" or t.name == "stableToken",
+      where: t.name in ^token_contract_names,
+      inner_join:
+        tkn in fragment(
+          """
+          (
+            WITH token_names AS (
+              SELECT contract_name, token_symbol FROM unnest(?::text[], ?::text[]) t (contract_name, token_symbol)
+            ) SELECT * FROM token_names
+          )
```

Comment:
Maybe these could be loaded from `tokens` table too (except that CELO still has `cGLD` as it's symbol in the contract)

Reply:
Do you think it’s safe to drop the join with CeloParams and join with Token instead? Do you know why we joined with CeloParams in the first place?

Reply:
The problem with using the token table is that there's no way to know which is the official cUSD token from just that table (somebody could create a token with same symbol and name)... So that's the reason why the params table has to be used sometimes (perhaps it would be faster store the params somewhere else though instead of the DB)